src/Bicep.Core/Workspaces/ISourceFile.cs

```diff
@@ -5,7 +5,8 @@ namespace Bicep.Core.Workspaces
 {
     public interface ISourceFile
     {
-        Uri FileUri { get; }
-        string GetOriginalSource();
+        Uri Identifier { get; }
```

Comment:
The next step is to update the type from `Uri` to `ResourceIdentifier`, which would be ... quite an interesting task.